restapi/configure_deathstar.go

```diff
@@ -84,7 +96,40 @@ func configureAPI(api *operations.DeathstarAPI) http.Handler {
 		return operations.NewPutExhaustPortServiceUnavailable().WithPayload(backtrace)
 	})
 	api.PostRequestLandingHandler = operations.PostRequestLandingHandlerFunc(func(params operations.PostRequestLandingParams) middleware.Responder {
-		return operations.NewPostRequestLandingOK().WithPayload("Ship landed\n")
+		var landingRequestURL, ok = os.LookupEnv("LANDING_REQUEST_URL")
+		if !ok {
+			return operations.NewPostRequestLandingOK().WithPayload(landingResponse() + "\n")
```

Comment:
I would have `landingResponse()` include the terminating `\n`. It already returns one of them, might as well add all of them in one place

Reply:
Also - and this is really not a big deal especially since it's demo code - for readability I'd have split this out into a named function, rather than have quite so many if / else indents. But really not a big deal